terraform/environments/corporate-staff-rostering/locals_security_groups.tf

```diff
@@ -546,6 +558,470 @@ locals {
+          description = "445: TCP SMB ingress from Azure DC"
+          from_port   = 445
+          to_port     = 445
+          protocol    = "TCP"
+          cidr_blocks = concat(local.security_group_cidrs.jumpservers, local.security_group_cidrs.domain_controllers)
+        }
+        smb_udp_domain = {
+          description = "445: UDP SMB ingress from Azure DC"
+          from_port   = 445
+          to_port     = 445
+          protocol    = "UDP"
+          cidr_blocks = concat(local.security_group_cidrs.jumpservers, local.security_group_cidrs.domain_controllers)
+        }
+        rpc_dynamic_udp_domain = {
+          description = "49152-65535: UDP Dynamic Port range"
+          from_port   = 49152
+          to_port     = 65535
+          protocol    = "UDP"
+          cidr_blocks = concat(local.security_group_cidrs.jumpservers, local.security_group_cidrs.domain_controllers)
+        }
+        rpc_dynamic_tcp_domain = {
+          description = "49152-65535: TCP Dynamic Port range"
+          from_port   = 49152
+          to_port     = 65535
+          protocol    = "TCP"
+          cidr_blocks = concat(local.security_group_cidrs.jumpservers, local.security_group_cidrs.domain_controllers)
+        }
+      }
+      egress = {
+        all = {
+          description = "Allow all traffic outbound"
+          from_port   = 0
+          to_port     = 0
+          protocol    = "-1"
+          cidr_blocks = ["0.0.0.0/0"]
+        }
+      }
+    }
+    database = {
+      description = "New security group for database servers"
+      ingress = {
+        all-from-self = {
+          description = "Allow all ingress to self"
+          from_port   = 0
+          to_port     = 0
+          protocol    = -1
+          self        = true
+        }
+        # IMPORTANT: check if an 'allow all from load-balancer' rule is required
+        echo_core_tcp_db = {
+          description = "7: Allow ingress from port 7 oem agent echo" # Not sure what this is
+          from_port   = 7
+          to_port     = 7
+          protocol    = "TCP"
+          cidr_blocks = local.security_group_cidrs.core
+        }
+        echo_core_udp_db = {
+          description = "7: Allow ingress from port 7 oem agent echo" # Not sure what this is
+          from_port   = 7
+          to_port     = 7
+          protocol    = "UDP"
+          cidr_blocks = local.security_group_cidrs.core
+        }
+        ssh-db = {
+          description = "22: SSH allow ingress"
+          from_port   = 22
+          to_port     = 22
+          protocol    = "tcp"
+          cidr_blocks = local.security_group_cidrs.ssh
+        }
+        rpc_udp_db = {
+          description = "135: UDP MS-RPC AD connect ingress from Azure DC and Jumpserver"
+          from_port   = 135
+          to_port     = 135
+          protocol    = "UDP"
+          cidr_blocks = concat(local.security_group_cidrs.jumpservers, local.security_group_cidrs.domain_controllers)
+        }
```

Comment:
I don't think this is needed as we're not joining the database to domain.

Reply:
Removed